scripts/09-history-migration-to-mongo.py

```diff
@@ -0,0 +1,455 @@
+#!/usr/bin/env python
+import json
+import logging
+import socket
+import sys
+import time
+
+import elliptics
+import msgpack
+import pymongo
+
+
+logger = logging.getLogger('mm.convert')
+
+CONFIG_PATH = '/etc/elliptics/mastermind.conf'
+
+try:
+
+    with open(CONFIG_PATH, 'r') as config_file:
+        config = json.load(config_file)
+
+except Exception as e:
+    raise ValueError('Failed to load config file %s: %s' % (CONFIG_PATH, e))
+
+
+def make_meta_session():
+    log = elliptics.Logger('/tmp/ell-namespace-convert.log', config["dnet_log_mask"])
+    node_config = elliptics.Config()
+    meta_node = elliptics.Node(log, node_config)
+
+    addresses = [elliptics.Address(host=str(node[0]), port=node[1], family=node[2])
+                 for node in config["metadata"]["nodes"]]
+    logger.info('Connecting to meta nodes: {0}'.format(config["metadata"]["nodes"]))
+    meta_wait_timeout = config['metadata'].get('wait_timeout', 5)
+
+    try:
+        meta_node.add_remotes(addresses)
+    except Exception as e:
+        logger.error('Failed to connect to any elliptics meta storage node: {0}'.format(
+            e))
+        raise ValueError('Failed to connect to any elliptics storage META node')
+
+    meta_session = elliptics.Session(meta_node)
+    meta_session.set_timeout(meta_wait_timeout)
+    meta_session.add_groups(list(config["metadata"]["groups"]))
+
+    time.sleep(5)
+
+    return meta_session
+
+
+def get_mongo_client():
+    if not config.get('metadata', {}).get('url', ''):
+        raise ValueError('Mongo db url is not set')
+    return pymongo.mongo_replica_set_client.MongoReplicaSetClient(config['metadata']['url'])
+
+
+def create_collection(db, coll_name):
+    try:
+        db.create_collection(coll_name)
+    except pymongo.errors.CollectionInvalid:
+        raise RuntimeError('Collection coll_name already exists')
+
+
+def create_indexes(coll):
+    coll.ensure_index([
```

Comment:
`ensure_index` is deprecated since version 3.0:
http://api.mongodb.org/python/current/api/pymongo/collection.html#pymongo.collection.Collection.ensure_index

What version of pymongo do you use?